rpm/harbour-cameraplus.spec

```diff
@@ -62,13 +70,19 @@ cp %SOURCE2 $RPM_BUILD_ROOT/usr/share/icons/hicolor/86x86/apps/
 # qtcamera configuration
 mkdir -p $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/
 mkdir -p $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/jolla/
-mkdir -p $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/nexus\ 5/
+ln -s jolla $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/jp-1301
```

Comment:
symlink is for backwards compatibility, for existing users who modified contents under jolla/ and are upgrading. I removed nexus 5 symlink, as that hasn't been publicly release yet, but for Jolla we need a symlink